Modules/posixmodule.c

```diff
@@ -9979,6 +9978,75 @@ os_copy_file_range_impl(PyObject *module, int src, int dst, Py_ssize_t count,
 }
 #endif /* HAVE_COPY_FILE_RANGE*/
 
+#ifdef HAVE_SPLICE
+/*[clinic input]
+
+os.splice
+    src: int
+        Source file descriptor.
+    dst: int
+        Destination file descriptor.
+    count: Py_ssize_t
```

Comment:
nit: Is the reason you called this `count` to keep consistency with `sendfile`? Might be better to call it `len` to keep consistent with linux man pages?

Reply:
I prefer to keep these names consistent with the rest of the API to reduce cognitive burden as we are already renaming the arguments everywhere